pallets/handles/src/handles-utils/src/tests/validator_tests.rs

```diff
@@ -70,3 +70,12 @@ fn test_consists_of_supported_unicode_character_sets_rejects_emojis() {
 
 	assert!(!consists_of_supported_unicode_character_sets(&string_containing_emojis));
 }
+
+#[test]
+fn test_confusables_map_does_not_contain_keys_in_unsupported_character_sets() {
+	for key in CONFUSABLES.keys() {
+		//TODO: Remove this println! when the test is passings
+		println!("codepoint: {:x}", *key as u32);
+		assert!(consists_of_supported_unicode_character_sets(&key.to_string()));
+	}
```

Comment:
This test will fail until until the CONFUSABLES map is regenerated to contain characters ONLY in supported ranges